gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -67,15 +70,17 @@ public fun GravatarQuickEditorBottomSheet(
     onAvatarSelected: () -> Unit,
     onDismiss: (dismissReason: GravatarQuickEditorDismissReason) -> Unit = {},
 ) {
-    GravatarQuickEditorBottomSheet(
-        gravatarQuickEditorParams = gravatarQuickEditorParams,
-        authenticationMethod = authenticationMethod,
-        onAvatarSelected = onAvatarSelected,
-        onDismiss = onDismiss,
-        modalBottomSheetState = rememberGravatarModalBottomSheetState(
-            avatarPickerContentLayout = gravatarQuickEditorParams.avatarPickerContentLayout,
-        ),
-    )
+    CompositionLocalProvider(LocalGravatarTheme provides mainGravatarTheme) {
```

Comment:
❓ I wonder if we should move this to the internal composable as it is also used by the `GravatarQuickEditorBottomSheetWrapper`. I mean, we are only applying the `CompositionLocalProvider` to the public composable but not to the one used by the old view system extensions. 

I've been trying to modify that in the test activity, and I couldn't, but it looks safer in the internal composable. Wdty?

Reply:
Oh yes, good catch!